meeshkan/__main__.py

```diff
@@ -69,30 +70,30 @@ def run_from_source(source: AbstractSource, sinks: Sequence[AbstractSink]):
 
 @click.command()
 @click.option("-i", "--input-file", required=False, type=click.File('rb'), help="Input file. Use dash '-' to read from stdin.")
+@click.option("-f", "--input-folder", required=False, type=click.Path(exists=True, file_okay=False, writable=True, readable=True), help="Input folder.")
 @click.option("-o", "--out", required=True, default='out', type=click.Path(exists=False, file_okay=False, writable=True, readable=True), help="Output directory. If the directory does not exist, it is created if the parent directory exists.")
-@click.option("--source", required=False, default='file', type=str, help="Source to read recordings from. For example, 'kafka'")
+@click.option("--kafka", required=False, default=False, type=bool, help="Read from kafka")
```

Comment:
The idea here was that `meeshkan` would be a generic stream processor where it's easy to add custom sources with stuff like `--source mymodule.blah:Source`. I know the thing's not very pretty at the moment but I would like to keep it easy to extend, to not run into the same problem as with `unmock-js`...

Reply:
If extensibility is the goal, then could we nix `--input` and any command-line arguments that are tightly coupled to a source? Also, what would this extension mechanism be? Even if it is not in existence yet, we can document the design so that contributors know how to add new sources.